tracing-log/src/trace_logger.rs

```diff
@@ -42,9 +43,9 @@ use tracing_core::{
 ///
 /// [`Subscriber`]: https://docs.rs/tracing/0.1.7/tracing/subscriber/trait.Subscriber.html
 /// [flags]: https://docs.rs/tracing/latest/tracing/#crate-feature-flags
-pub struct TraceLogger {
+pub struct TraceLogger<'a> {
```

Comment:
This type is deprecated and is going to be removed in the next release, so you don't really need to bother updating it --- we'll just remove it entirely. Maybe open a separate PR for that, and rebase this one onto it, if you don't mind?